containers/azure-terraform/.devcontainer/devcontainer.env

```diff
@@ -0,0 +1,20 @@
+# the target azure subscription
+ARM_SUBSCRIPTION_ID=
+
+# the name of the resource group where the storage account for the backend should be created
+TF_BACKEND_RESOURCE_GROUP=
+
+# the region where the resource group for the backend should be created
+TF_BACKEND_LOCATION=
+
+# the name of the storage account for the backend
+TF_BACKEND_STORAGE_ACCOUNT=
+
+# the name of the storage account container for the backend
+TF_BACKEND_CONTAINER=
+
+# the name of the storage account container blob for the backend
+TF_BACKEND_KEY=
+
+# the resource group name to be provisioned with terraform
+TF_VAR_RG_NAME=
```

Comment:
We should remove any var related to the sample.

Reply:
Please see above clarification, thank you.